fullstop-plugins/fullstop-snapshot-source-plugin/src/main/java/org/zalando/stups/fullstop/plugin/snapshot/SnapshotSourcePlugin.java

```diff
@@ -78,18 +80,17 @@ public void processEvent(CloudTrailEvent event) {
 
             if (userData == null) {
                 violationSink.put(violationFor(event).withInstanceId(id).withPluginFullyQualifiedClassName(
-                                          SnapshotSourcePlugin.class).withType(MISSING_USER_DATA).build());
+                        SnapshotSourcePlugin.class).withType(MISSING_USER_DATA).build());
                 return;
             }
 
             String source = (String) userData.get(SOURCE);
             if (source == null) {
                 violationSink.put(violationFor(event).withInstanceId(id).withPluginFullyQualifiedClassName(
-                                          SnapshotSourcePlugin.class).withType(MISSING_SOURCE_IN_USER_DATA).build());
-            }
-            else if (source.matches(SNAPSHOT_REGEX)) {
+                        SnapshotSourcePlugin.class).withType(MISSING_SOURCE_IN_USER_DATA).build());
+            } else if (source.matches(SNAPSHOT_REGEX)) {
                 violationSink.put(violationFor(event).withInstanceId(id).withPluginFullyQualifiedClassName(
-                                          SnapshotSourcePlugin.class).withType(EC2_WITH_A_SNAPSHOT_IMAGE).build());
+                        SnapshotSourcePlugin.class).withType(EC2_WITH_A_SNAPSHOT_IMAGE).withMetaInfo(singletonMap("application", source)).build());
```

Comment:
let's call it "deployment_artifact" instead of "application" to fit our new wording guidelines.